en/docs/learn/api-security-configuration.md

```
@@ -10,6 +10,27 @@ name = "com.wso2.openbanking.accelerator.gateway.executor.impl.mtls.cert.validat
 priority = 1
 ``` 
 
+###Configuring MTLS when TLS is terminated at the load balancer (APIM)
```

markdownlint-cli2 (0.22.1)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)